benchmarks/Cargo.toml

```diff
@@ -41,13 +41,13 @@ mimalloc = { version = "0.1", optional = true, default-features = false }
 num_cpus = "1.13.0"
 object_store = "0.5.0"
 parquet = "25.0.0"
-parquet-test-utils = { path = "../parquet-test-utils/" }
+parquet-test-utils = { path = "../parquet-test-utils" }
```

Comment:
Are these new dependencies only used in tests? If so, maybe they can move to `[dev-dependencies]`?

Reply:
I just took away the slash. I didn't add these. I can put the slash back if that's better form.